### ISSUE TITLE
Add get/setBigInteger to IntegerType?

### ISSUE THREAD

**Issue by awalter17** [imglib/imglib2#105]
Hello! 

Currently the [IntegerType](https://github.com/imglib/imglib2/blob/master/src/main/java/net/imglib2/type/numeric/IntegerType.java) interface has getters and setters only for `long` and `int` values. However, this interface is implemented by [Unsigned128BitType](https://github.com/imglib/imglib2/blob/master/src/main/java/net/imglib2/type/numeric/integer/Unsigned128BitType.java) which is backed by a `BigInteger`. This causes an issue if someone ever wants to get the value of an IntegerType. 

```
public <T extends IntegerType<T>> void printValue(final T input) {
    System.out.println(input.getIntegerLong());
}
```

For example, in the above method if `input` turned out to be an Unsigned128BitType with a value greater than 2<sup>63</sup> - 1, I could never get the actual value of that Unsigned128BitType without knowing it is an Unsigned128BitType. 

As such, I think it would be beneficial to add the methods `getBigInteger()` and `setBigInteger()`, to the IntegerType interface. 

Please let me know your thoughts on this, and thank you in advance for your input.

**Comment by tpietzsch** [comment]
@awalter17 I agree. It would be good to add those

**Comment by ctrueden** [comment]
Addressed by #109.
